control_plane/src/background_process.rs

```diff
@@ -36,11 +36,12 @@ use utils::pid_file::{self, PidFileRead};
 // it's waiting. If the process hasn't started/stopped after 5 seconds,
 // it prints a notice that it's taking long, but keeps waiting.
 //
-const RETRY_UNTIL_SECS: u64 = 10;
-const RETRIES: u64 = (RETRY_UNTIL_SECS * 1000) / RETRY_INTERVAL_MILLIS;
-const RETRY_INTERVAL_MILLIS: u64 = 100;
-const DOT_EVERY_RETRIES: u64 = 10;
-const NOTICE_AFTER_RETRIES: u64 = 50;
+const DEFAULT_STOP_RETRY_TIMEOUT: Duration = Duration::from_secs(10);
+const DEFAULT_STOP_RETRIES: u128 =
```

Comment:
nit: (possibly I suggested the wrong thing earlier): there is no way to change stop retry count or timeout, so, the `DEFAULT_` prefix is redundant / misleading.